requirements/dev.txt

```diff
@@ -13,7 +13,7 @@ attrs==22.1.0
     # via
     #   -r requirements/needle.txt
     #   pytest
-build==0.8.0
+build==0.9.0
```

Comment:
No breaking change mentioned in the new version https://pypa-build.readthedocs.io/en/stable/changelog.html#id1